src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import { Movie } from '../../app.typedefs';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  movies: Movie[];
+}
```

Comment:
so `MoviesList` component expects prop `movies` to be passed
but `App.tsx` does not pass anything =..(